include/bb2diaspora.php

```diff
@@ -7,6 +7,29 @@
 require_once("include/bbcode.php");
 require_once("library/html-to-markdown/HTML_To_Markdown.php");
 
+/**
+ * @brief Callback function to replace a Diaspora style mention in a mention for Friendica
+ *
+ * @param array $match Matching values for the callback
+ * @return text Replaced mention
+ */
+function diaspora_mention2bb($match) {
+	if ($match[2] == '') {
+		return;
+	}
+
+	/// @todo Don't use probe
```

Comment:
since the probe part is commented out it isn't a todo anymore :-) Please delete `/ @todo` so it would be a 'normal' comment